modules/apps/27-interchain-accounts/types/codec_test.go

```diff
@@ -300,40 +298,24 @@ func (suite *TypesTestSuite) TestSerializeAndDeserializeCosmosTx() {
 		}
 
 		// test serializing non sdk.Msg type
-		bz, err := types.SerializeCosmosTx(simapp.MakeTestEncodingConfig().Codec, []proto.Message{&banktypes.MsgSendResponse{}}, encoding)
+		bz, err := types.SerializeCosmosTx(suite.chainA.Codec, []proto.Message{&banktypes.MsgSendResponse{}}, encoding)
 		suite.Require().NoError(err)
 		suite.Require().NotEmpty(bz)
 
 		// test deserializing unknown bytes
-		msgs, err := types.DeserializeCosmosTx(simapp.MakeTestEncodingConfig().Codec, bz, encoding)
+		msgs, err := types.DeserializeCosmosTx(suite.chainA.Codec, bz, encoding)
 		suite.Require().Error(err) // unregistered type
 		suite.Require().Contains(err.Error(), expDeserializeErrorStrings[i])
 		suite.Require().Empty(msgs)
 
 		// test deserializing unknown bytes
-		msgs, err = types.DeserializeCosmosTx(simapp.MakeTestEncodingConfig().Codec, []byte("invalid"), encoding)
+		msgs, err = types.DeserializeCosmosTx(suite.chainA.Codec, []byte("invalid"), encoding)
 		suite.Require().Error(err)
 		suite.Require().Contains(err.Error(), expDeserializeErrorStrings[i])
 		suite.Require().Empty(msgs)
 	}
 }
 
-// unregistered bytes causes amino to panic.
-// test that DeserializeCosmosTx gracefully returns an error on
-// unsupported amino codec.
-func (suite *TypesTestSuite) TestProtoDeserializeAndSerializeCosmosTxWithAmino() {
```

Comment:
amino codec no longer is supported so it doesn't implement the required interface for usage (it doesn't fulfill the interface of the `SerializeCosmosTx` function